packages/react/src/exampleTypes/ExpandableTagsQuery/index.js

```diff
@@ -132,7 +136,11 @@ let TagsWrapper = observer(
     popoverOffsetY,
     theme: { Icon, TagsInput, Tag, Popover },
     joinOptions,
-    wordsMatchPattern,
+    // Allow tag keywords to contain dots in them if the user is searching for exact
+    // words instead of their variations.
+    wordsMatchPattern = node.exact
+      ? alphaNumericRegExWithDots
+      : alphaNumericRegEx,
```

Comment:
Why would not always use `alphaNumericRegExWithDots`? Also, why are the regexs so strange looking?

Reply:
I thought that's we had agreed on, that if you didn't search for exact you'd get the current behavior?

I think yhey are strange looking because they were copied from lodash. See [this comment](https://github.com/smartprocure/contexture/pull/203/files#diff-431701c69ca84ff7bf09ad61f1da89a10d000c32116db6b3795354611d40de6eR40)